internal/cli/cmd/cmd.go

```diff
@@ -84,5 +84,14 @@ cilium connectivity test`,
 	cmd.SetOut(os.Stdout)
 	cmd.SetErr(os.Stderr)
 
+	cmd.AddCommand(&cobra.Command{
+		Use:     "bpf",
+		Aliases: []string{"cleanup", "debuginfo", "endpoint", "fqdn", "identity", "ip", "kvstore", "lrp", "map", "metrics", "monitor", "node", "policy", "prefilter", "preflight", "recorder", "service"},
+		Hidden:  true,
+		Long: `Exec into the daemonset to use this subcommand.
```

Comment:
Wording suggestion:
"This command is only available in the Cilium agent.
Exec in to a running cilium pod to use it.
kubectl exec -it -n kube-system ds/cilium -- bash`,